src/modules/renderGraphiQL.ts

```diff
@@ -0,0 +1,149 @@
+/*
+ * Mostly taken straight from express-graphql, so see their licence
+ * (https://github.com/graphql/express-graphql/blob/master/LICENSE)
+ */
+
+// TODO: in the future, build the GraphiQL app on the server, so it does not
+// depend on any CDN and can be run offline.
+
+export type GraphiQLData = {
```

Comment:
we should add comments for each property of graphiqldata to make it easier to understand the propose of each option.   agree?